components/Common/Pagination/useGetPageElements.tsx

```diff
@@ -0,0 +1,146 @@
+'use client';
```

Comment:
I don't think we should add `'use client';` at this point of time

Reply:
Okay, thank you. I will remove these